tests/integration/common_workflows/test_sql_asset_workflows.py

```diff
@@ -109,6 +109,55 @@ def _create_test_cases():
     ]
 
 
+def test_get_batch_identifiers_list__simple(
+    postgres_daily_batch_definition: BatchDefinition,
+) -> None:
+    batch_definition = postgres_daily_batch_definition
+    batch_identifiers_list = batch_definition.get_batch_identifiers_list()
+
+    assert len(batch_identifiers_list) == 10
+    # just spot check the edges
+    assert batch_identifiers_list[0] == {"year": None, "month": None, "day": None}
+    assert batch_identifiers_list[-1] == {"year": 2001, "month": 1, "day": 1}
```

Comment:
Why are the values `None` for this first one?

Reply:
There was one entry in the sample data where those fields are None, and when values are missing, they come first when sorting ascending.